src/Types.hs

```diff
@@ -803,17 +828,20 @@ mhKeepTemp = lens _mhKeepTemp (\h x -> h { _mhKeepTemp = x })
 -- exits.
 data EntityCommand m a = EntityCommand
```

Comment:
This feels messy. I guess I still have a bit of time until a new brick release comes out to find perhaps a better solution.